assets/js/custom.js

```diff
@@ -1 +1,29 @@
 // Put your custom JS code here
+
+// self-hosting view.
+
+const storedSelfhost = localStorage.getItem('selfhost');
+
+const getSelfhost = () => {
+  if (storedSelfhost) {
+    return storedSelfhost;
+  }
+
+  return 'no';
+};
+
+const setSelfhost = function (selfhost) {
+  document.documentElement.setAttribute('data-bs-selfhost', selfhost);
```

Comment:
Could we call this data attribute `data-selfhost` (bs = Bootstrap, which this seems orthogonal to?)

Reply:
Absolutely, this is a remnant from the initial phase where I took strong inspiration from the already existing code in doks.